hack/windows/Prepare-Node.ps1

```diff
@@ -125,7 +125,9 @@ if ($InstallKubeProxy) {
 
 $StartKubeletFileContent += [Environment]::NewLine + '$global:KubeletArgs += "--cert-dir=$env:SYSTEMDRIVE\var\lib\kubelet\pki --config=/var/lib/kubelet/config.yaml --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"mcr.microsoft.com/oss/kubernetes/pause:1.4.1`" --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`" --resolv-conf=`"`" --node-ip=$env:NODE_IP"'
 
-if ($ContainerRuntime -eq "containerd") {
+$targetVersion = [version]"1.28.0"
```

Comment:
Does this mean that installing versions after 1.28.0, such as 1.28.1, will still include the `WindowsHostProcessContainers`? @NamanAg30

Reply:
No we will have this flag only for versions less than k8s 1.28.0
I have used less than condition in the next line for comparison of versions.
[version]($KubernetesVersion -replace '^v') -lt $targetVersion

Reply:
Got it, thanks.